gateway.distribution.version/pom.xml

```diff
@@ -1,85 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does removing this affect what the management console displays for the Gateway version?